src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/mgr-modules/mgr-module-list/mgr-module-list.component.ts

```diff
@@ -88,7 +88,7 @@ export class MgrModuleListComponent extends ListWithDetails {
         name: $localize`Disable`,
         permission: 'update',
         click: () => this.updateModuleState(),
-        disable: () => () => this.getTableActionDisabledDesc(),
+        disable: () => this.getTableActionDisabledDesc(),
```

Comment:
The changes in this file are also contained in this merged PR:
https://github.com/ceph/ceph/pull/37753

Maybe you need to rebase?